### PR TITLE
Mat 6463 update madie translator and cq framework

### PR DESCRIPTION
**Pull request by gregory-akins**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-6463](https://jira.cms.gov/browse/MAT-6463)
(Optional) Related Tickets:

### All Submissions
* [X] This PR has the JIRA linked.
* [X] Required tests are included.
* [X] No extemporaneous files are included (i.e Complied files or testing results).
* [X] This PR is merging into the **correct branch**.
* [X] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/mat/cql_elm_translation/service/EffectiveDataRequirementService.java [hunk 1]
```diff
@@ -36,45 +29,6 @@ private RequestData createDefaultRequestData(String cql) {
         .build();
   }
 
-  /**
-   * @param libraryDetails - instance of CqlLibraryDetails
-   * @param recursive- indicates if the data requirements gathered should be recursive
-   * @param accessToken- used by MadieLibrarySourceProvider to make calls to cql-library-services
-   * @return effective data requirement of type R5 library
-   */
-  public org.hl7.fhir.r5.model.Library getEffectiveDataRequirements(
```
Comment (adongare) [inline src/main/java/gov/cms/mat/cql_elm_translation/service/EffectiveDataRequirementService.java:45]: we can delete this service and its unit test completely. this is specific to FHIR/QICore

**Review by adongare — APPROVED** [review]
